src/test/scala/org/hammerlab/guacamole/main/SomaticJointCallerIntegrationTests.scala

```diff
@@ -95,7 +95,7 @@ object SomaticJointCallerIntegrationTests extends SparkCommand[Arguments] with V
         args.out = resultFile
         args.paths = Seq(NA12878TestUtil.subsetBam).toArray
         args.loci = "chr1:0-6700000"
-        args.forceCallLociFromFile = NA12878TestUtil.expectedCallsVCF
+        args.forceCallLociFile = NA12878TestUtil.expectedCallsVCF
```

Comment:
This test doesn't run as part of Travis / `mvn test`; cf. recent discussion at https://github.com/hammerlab/guacamole/issues/483#issuecomment-231597188.

Might make sense to have a test of the loci-from-VCF functionality though; I'll add that to this PR.